src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -413,5 +413,32 @@ public float Volume
                     ThrowIfFailed(this, "Failed to set the volume of the player");
             }
         }
+
+        /// <summary>
+        /// Gets or sets the audio-only state.
+        /// </summary>
+        /// <value>true if the playback is audio-only mode; otherwise, false. The default value is false.</value>
+        /// The <see cref="Player"/> To set or get, the player must be in the <see cref="PlayerState.Ready"/>,
```

Comment:
How about below?
/// The `<see cref="Player"/>` must be in th the ~~

Reply:
ok